src/DotLiquid/ShopifyFilters.cs

```diff
@@ -0,0 +1,97 @@
+using System.Globalization;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace DotLiquid
+{
+    /// <summary>
+    /// Shopify filters implemented by DotLiquid
+    /// </summary>
+    /// <see href="https://shopify.dev/docs/themes/liquid/reference/filters"/>
+    public static class ShopifyFilters
+    {
+        /// <summary>
+        /// Converts a string into an MD5 hash.
+        /// </summary>
+        /// <see href="https://shopify.dev/docs/themes/liquid/reference/filters/string-filters#md5"/>
+        /// <param name="input"/>
+        public static string Md5(string input)
+        {
+            return input.IsNullOrWhiteSpace() ? input : ComputeHash(MD5.Create(), input);
```

Comment:
`HashAlgorithm : IDisposable, `
so you always need to dispose the classes that you pass in to this helper
At least for .NET Core
https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.hashalgorithm?view=net-5.0

Reply:
Fixed.